content/cli/commands/profile.md

```diff
@@ -2,3 +2,161 @@
 title: Profile
 weight: 50
 ---
+
+Fluvio provides support for multiple profiles out of the box, meaning that you can
+communicate with distinct clusters in an isolated and managed way. Each profile
```

Comment:
A profile is a collection of settings and credential that you can apply to a Fluvio session.